etc/telegraf.conf

```diff
@@ -6937,8 +6937,17 @@
 #   ## Content encoding for message payloads, can be set to "gzip" to or
 #   ## "identity" to apply no encoding.
 #   # content_encoding = "identity"
-
-
+#
+#
```

Comment:
No hashes here, was correct before. Furthermore, this file needs no update as it is automatically updated at release time IIRC.

Reply:
I reverted the etc/telegraf.conf changes